nlu.py

```diff
@@ -1,5 +1,28 @@
 import jieba
 
+command_mapping = {'曠課': '昌昌;曠課', '課表': '昌昌;課表', '成績': '昌昌;成績'}
+
+
 def response_message(text: str) -> str:
-    split_text = jieba.cut(text)
-    return ", ".join(split_text)
+    cmds = _get_commands(jieba.cut(text))
+    return _exec_command(cmds)
+
+
+def _get_commands(text_sequence: [str]) -> set:
+    commands = set()
+    for text in text_sequence:
+        if text in command_mapping:
+            commands.add(command_mapping.get(text))
+    return commands
+
+
+def _exec_command(cmds: set) -> str:
```

Comment:
If you send a `set` into the function, then it should be named `_exec_commands`.

For me, why not just keep original behavior, but iterate the `set` to get all commands' result.

Then later we could have a chance to decide how to format the result into the response(grouping it or multiple-response)

Reply:
You're right, iterate the set it more readable. Let's keep simple and stupid.